packages/components/src/ListReview/components/Row.tsx

```diff
@@ -17,19 +17,21 @@ export const RowContainer = styled.tr`
   td {
     padding: 14px 0;
     vertical-align: top;
+    word-wrap: anywhere;
+    white-space: normal;
   }
 
   td:first-child {
     margin: 0;
     ${({ theme }) => theme.fonts.bold16}
-    min-width: 300px;
+    width: 300px;
```

Comment:
Can nth-child and nth-child(2) - not be 100%? so that they equally share the remaining space

eg. available width for columns 1 and 2 is 620px. Each has padding-right as 8px.
So each column is 312px wide

Reply:
under mediia query max-width 768px to 577px, this is done, since 1st and 2nd row is stacked, it is possible. But media query above 768, this does not work.